src/hazelweb/cardstacks/builtins/TutorialCards.re

```diff
@@ -106,9 +101,4 @@ let intro_card: CardInfo.t = {
    caption: empty_hole_insertion_caption,
    init_zexp: empty_hole_insertion_init_zblock,
  };
- */
-
-let cardstack: CardstackInfo.t = {
-  title: "Hazel Tutorial",
-  cards: [intro_card /*backspace_card, empty_hole_insertion_card */],
-};
+ */
```

Comment:
this whole file isn't usable anymore without the cardstack definition. I'm fine with just removing the file entirely -- not much to salvage here.

Reply:
sounds good! I removed the old code